### PR TITLE
Login signup page

### PR DESCRIPTION
**Pull request by Ananyamadhu08**
- created login page 
- created logout page

### CODE REVIEW

**Review by j24m — APPROVED** [review]
Hello, the markdown for login signup looks fine to me, didn;t see anything that needs changes. Good work. :)

**Review by Debarshi95 — APPROVED** [review]
Looks good. Can merge it.

File: index.css [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+}
+.search__bar:focus:-ms-placeholder {
+  transition: opacity 0.45s ease;
+  opacity: 0;
+}
+
+.search__icon {
+  position: relative;
+  left: -30px;
+}
```
Comment (mayuresh-tech) [inline index.css:42]: You can use rem
And -30 doesn't look good

File: index.html [hunk 2]
```diff
@@ -0,0 +1,314 @@
...
+    />
+  </head>
+  <body class="bg-grey-100">
+    <!-- header -->
+    <header class="w-full h-20 shadow bg-slate-900 fixed z-50">
+      <div class="flex justify-around align-items-center h-full w-full">
+        <!-- header-left (logo, nav-links) -->
+        <div class="flex justify-center align-items-center">
+          <!-- Logo -->
+          <a href="/index.html">
```
Comment (mayuresh-tech) [inline index.html:33]: Nice comments!

**Review by mayuresh-tech — APPROVED** [review]
Good to merge!

File: index.css (hunk 1 above)
Comment (Ananyamadhu08) [reply index.css:42]: okay will make the changes

File: index.css [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+  opacity: 0;
+}
+.search__bar:focus:-ms-placeholder {
+  transition: opacity 0.45s ease;
+  opacity: 0;
+}
+
+.search__icon {
+  position: relative;
+  left: -30px;
```
Comment (harshkotkar5557) [inline index.css:41]: use of % or rem instead of pixles

File: index.html [hunk 4]
```diff
@@ -0,0 +1,315 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Fusion E-com</title>
+    <link
+      href="https://fusion-css.netlify.app/css/index.css"
+      rel="stylesheet"
+    />
+    <link href="index.css" rel="stylesheet" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.2/css/all.min.css"
```
Comment (harshkotkar5557) [inline index.html:15]: You can put CDN in directly you main css file instead of adding in every html file.

**Review by harshkotkar5557 — COMMENTED** [review]
clean code though I have pointed some comments

**Pull request merged**